Symfony/CS/Tests/UtilsTest.php

```diff
@@ -17,21 +17,24 @@
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  * @author Graham Campbell <graham@mineuk.com>
+ * @author Odín del Río <odin.drp@gmail.com>
  */
 class UtilsTest extends \PHPUnit_Framework_TestCase
 {
     /**
      * @dataProvider provideCamelCaseToUnderscoreCases
+     *
+     * @param string $expected Camel case string.
+     * @param string $input    Input string.
      */
-    public function testCamelCaseToUnderscore($expected, $input = null)
+    public function testCamelCaseToUnderscore($expected, $input)
```

Comment:
Yes but, that the condition ```if (null !== $input)``` wasn't satisfied for any of the provided examples (and there was only one), so, I think that is better to remove that unused code.
Also, I added the case with the same input and expected output in order to test that an already underscore string doesn't change. In my opinion, is better to duplicate the input as the expected output in the provider instead of has an optional parameter, an if, and a double assert in the same test method.

Another way may be add another test method to test strings that should not be modified by the ```Utils::camelCaseToUnderscore```.

what do you think @keradus?